Solutions/IoTOTThreatMonitoringwithDefenderforIoT/Playbooks/AutoCloseIncidents/AutoCloseIncidents.json

```diff
@@ -230,7 +230,12 @@
                      "connectionId":"[resourceId('Microsoft.Web/connections', variables('AzureSentinelConnectionName'))]",
                      "connectionName":"[variables('AzureSentinelConnectionName')]",
                      "id":"[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/azuresentinel')]"
-                  }
+                  },
```

Comment:
This is hardcoded. Can you please use the concat function as in line number 232.
Thanks